### DISCUSSION TITLE
Does the script work with Endeavour OS?

### DISCUSSION THREAD

**Discussion by ejsadiarin** [prasanthrangan/hyprdots#40] [Q&A]
I tried but it didn't work (it stopped when creating dir for grub, permission denied), i think its because my default bootloader that time was systemd. But I'm going to try tomorrow to see if it works.

is it also recommended to `sudo ./install.sh` before running the script?

**Comment by prasanthrangan** [comment]
Please DO NOT execute the install script as sudo.
It should prompt you to input sudo password while creating the grub dir, and your user should also have sudo privileges.

**Reply by ejsadiarin** [reply]
Oh ok got it!
